stock_storage_type/tests/test_storage_type_move.py

```diff
@@ -371,5 +371,3 @@ def _levels_for(packages):
                 move_line.qty_done = move_line.product_uom_qty
 
         second_level.location_dest_id = third_level.location_dest_id
-        with self.assertRaises(ValidationError):
```

Comment:
FIXME This error is not raised anymore but the contraint _check_storage_type is still there

Reply:
I put back this check. However, I don't get why the constrains is not loaded by the ORM when I run the test locally